# HG changeset patch
# User jwe@segfault.lan
# Date 1201925192 18000
#      Fri Feb 01 23:06:32 2008 -0500
# Node ID e010120ba03e0584aa5351228c4d6fb558b7eac0
# Parent  10da0f6d85c260bb5f640436e42d633a57ccda9b
Added tag ss-2-9-11 for changeset 10da0f6d85c2

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -21,8 +21,9 @@ f16c05db625059770b94eec0af027bbe74d2ce62
 cda6a105ae9a30f9cd8032b1f49c2cfecfc04f6c before-ov-branch
 acb4a1e0b311c47bc957a48bf6af7f77e9c80fab ss-2-9-6
 b2e1be30c8e94902e6557c1edba2c4f37ff7bdf8 ss-2-9-7
 d542d919783961dce51ee8fc06f65b8f2cfe4156 ss-2-9-8
 60f9ced8ab534e65f27d95a712d97ba6add6cd8d ss-2-9-9
 12b676a0b18395af2fa0ccceb516447d8be6ad03 before-graphics-branch
 a848b846cb3ac279782b7fef8270a62ca4578539 ss-2-9-10
 3e2cfed6c87e2019cb1584211068b9c581331ccb before-object-branch
+10da0f6d85c260bb5f640436e42d633a57ccda9b ss-2-9-11
